Userland/Libraries/LibCrypto/Cipher/AES.cpp

```diff
@@ -374,6 +675,34 @@ void AESCipher::decrypt_block(AESCipherBlock const& in, AESCipherBlock& out)
     // clang-format on
 }
 
+#if (ARCH(I386) || ARCH(X86_64))
+static void __attribute__((target("aes"), used)) AESCipher_decrypt_block(AESCipherKey const& key, AESCipherBlock const& in, AESCipherBlock& out)
+{
```

Comment:
maybe move this up so that this is in the same big `#if X86` block

Reply:
I tried to keep the encrypt-portable and encrypt-accelerated functions close together, same for decrypt-portable and decrypt-accelerated, key-expand-portable and key-expand-accelerated and so on. Other ordering would be to do all the portable functions followed by all the accelerated functions. Which ordering are you preffering?